src/types/date-picker.ts

```diff
@@ -0,0 +1,16 @@
+export enum DatePickerMode {
+  DATE = 'date',
+  DATETIME = 'datetime',
+  TIME = 'time',
+}
+
+export enum PlatformType {
```

Comment:
Does RN TS does not expose something similar already?

It feels odd to have this typedef in `date-picker` as its not related to it at all.